src/model/dfcxx/include/dfcxx/typedefs.h

```diff
@@ -61,7 +61,9 @@ enum Scheduler {
 
 typedef std::unordered_map<dfcxx::Ops, unsigned> DFLatencyConfig;
 
-// Macro substitutions for indexes in a std::vector for paths.
+// Has to be incremented with every new output format.
+#define OUTPUT_FORMATS_COUNT 1
+// Macro substitutions for indexes in a std::vector for output paths.

```

Comment:
This comment is too code-dependent. What do you mean, when you say "std::vector for output paths"? Is it ok, that the comment become outdated if the specified type will be changed to something else?

So, let's rephrase it into something more general and understandable.

Reply:
Tried to fix this.